apps/opencs/model/world/refidcollection.hpp

```diff
@@ -44,7 +46,7 @@ namespace CSMWorld
 
         private:
 
-            const RefIdAdapter& findAdaptor (UniversalId::Type) const;
+            /*const*/ RefIdAdapter& findAdapter (UniversalId::Type) const;
```

Comment:
Problematic. The const is there for a reason; namely that adapters don't hold document state and therefore should not be modified.